src/services/cache/CacheService.ts

```diff
@@ -70,6 +74,15 @@ export class CacheService {
     return deferred.promise;
   }
 
+  getCacheUsingType(name: string, type: string, container: string, profile?: string): ng.IPromise<ICache> {
```

Comment:
It seems that `getCache(..)` is now redundant (I couldn't find any usages and I think it's broken anyway). So we should remove `getCache(..)` all together and maybe rename this to `getCache()`